docs/source/using_gridappsd/test_manager.rst

```diff
@@ -0,0 +1,134 @@
+
+1. Test Configuration
+The test configuration contains information to define the simulation run,
```

Comment:
Add a space between Test Configuration and "The test" so that a hard return will show up in the built docs.  I am in the process of having documentation on how to build the docs through a docker image at present so that will be easier to test soon.

Reply:
I will add a line.